src/acp-agent.ts

```diff
@@ -75,6 +77,16 @@ type BackgroundTerminal =
       pendingOutput: TerminalOutputResponse;
     };
 
+/**
+ * Extra metadata that the agent provides for each tool_call / tool_update update, under the `_meta.claudeCode` key.
+ */
+export type ToolUpdateMeta = {
+  /* The name of the tool that was used in Claude Code. */
+  toolName: string;
```

Comment:
Nice this is on the list of things to add to tool calls in general. Meta makes sense for now